bftengine/src/bftengine/ControllerWithSimpleHistory.hpp

```diff
@@ -54,12 +54,17 @@ class ControllerWithSimpleHistory : public ControllerBase {
 
   // Sets PrePrepare sending timepoint
   virtual void onSendingPrePrepare(SeqNum n, CommitPath commitPath) override;
+  void onSendingPrePrepare(SeqNum n, CommitPath commitPath, const Time& timePoint);
 
   // Marks request that was downgraded to slow path
   virtual void onStartingSlowCommit(SeqNum n) override;
 
   // Adds replica to a set of replicas, that have replied to a PrePrepare msg with a corresponding PreparePartialMsg.
   virtual void onMessage(const PreparePartialMsg* m) override;
+  //-- events
```

Comment:
what does `//-- events` mean?

Reply:
End of events methods section, I'll make it clearer